src/app/view/components/_common/dot-wizard/dot-wizard.component.html

```diff
@@ -0,0 +1,17 @@
+<dot-dialog
+    #dialog
+    [visible]="steps && steps.length"
+    [actions]="dialogActions"
+    (hide)="close()"
+    >
+    <div class="dot-wizard__view" *ngIf="steps && steps.length">
```

Comment:
I think that now you can do: 

```suggestion
    <div class="dot-wizard__view" *ngIf="steps?.length">
```